docs/getting-started/kubernetes/kind/index.md

```diff
@@ -0,0 +1,619 @@
+# Getting Started with Kind
+
```

Comment:
You should give an overview of what this doc is about similar to the other quick start guides. ".... setup argocd-agent using `kind` with a link to the kind website.